Kernel/Graphics/VirtIOGPU/Console.h

```diff
@@ -7,7 +7,7 @@
 #pragma once
 
 #include <Kernel/Graphics/Console/GenericFramebufferConsole.h>
-#include <Kernel/Graphics/VirtIOGPU/FramebufferDevice.h>
+#include <Kernel/Graphics/VirtIOGPU/DisplayConnector.h>
 #include <Kernel/TimerQueue.h>
 
 namespace Kernel::Graphics::VirtIOGPU {
```

Comment:
Since you've put both VirtIODisplayConnector and VirtIOGraphicsAdapter directly in the Kernel namespace, it might be more consistent to name this console VirtIOConsole and put it in the Kernel namespace as well, instead of keeping it in `Kernel::Graphics::VirtIOGPU::Console`.